eng/scripts/Create-Package-WorkItem.ps1

```diff
@@ -0,0 +1,98 @@
+[CmdletBinding()]
+param (
+  [Parameter(Mandatory = $true)]
+  [string]$Language,
+  [Parameter(Mandatory = $true)]
+  [string]$ServiceName,
+  [Parameter(Mandatory = $true)]
+  [string]$PackageName,
+  [Parameter(Mandatory = $true)]
+  [string]$PackageDisplayName,
+  [Parameter(Mandatory = $true)]
+  [string]$PackageType,
+  [Parameter(Mandatory = $true)]
+  [string]$PackageVersion,
+  [Parameter(Mandatory = $true)]
+  [string]$ReleaseDate,
+  [string]$RelatedWorkItemId,
+  [string]$Tag = "",
+  [string]$WorkingDir = ".",
+  [string]$PackageRootPath = "",
+  [string]$Devops_pat = $env:DEVOPS_PAT
+)
+
+Set-StrictMode -Version 3
+
+function Get-Repo-Name($language)
+{
+    switch ($language)
+    {
+        ".NET" { return "azure-sdk-for-net" }
+        "Python" { return "azure-sdk-for-python" }
+        "Java" { return "azure-sdk-for-java" }
+        "JavaScript" { return "azure-sdk-for-js" }
+        "Go" { return "azure-sdk-for-go" }
+        "C" { return "azure-sdk-for-c" }
+        "C++" { return "azure-sdk-for-cpp" }
+    }
+    Write-Host "Unknown language to map it to repo name. Language: [$language]"
+    return ""
+}
+
+function Clone-Repo($repoName, $targetDir)
+{
+    git clone "https://github.com/azure/$repoName.git"  $targetDir
```

Comment:
Might not be necessary but we should consider a shallow clone to speed things up.

Reply:
We don't know the service directory to sparse checkout only the package path. Are you suggesting to skip any test directories? I can update the depth to make it as shallow clone.

Reply:
I was just meaning to we could do a sparse clone which doesn't include any history which reduces the cost of cloning. Either way it is just an optimization.